pitch.md

```diff
@@ -57,13 +125,38 @@ the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+This project is pretty large. I feel like there is a huge amount of work to be
+done in language design. Figuring out how to integrate lots of different methods
+of robot control and allow users to easily specify logic without limiting them
```

Comment:
Ahh I see. I wasn't thinking in terms of an internal language. In that case, the only way I can think of to limit the user in an internal language (since the user would still have access to all the features of the host language) is to override an operation in the host language.